cime_config/allactive/config_compsets.xml

```diff
@@ -14,7 +14,7 @@
     TIME_ATM[%phys]_LND[%phys]_ICE[%phys]_OCN[%phys]_ROF[%phys]_GLC[%phys]_WAV[%phys][_ESP%phys][_BGC%phys]
     Where for the EAM specific compsets below the following is supported
     TIME = Time period (e.g. 2000, HIST, RCP8...)
-    ATM  = [EAM, SATM]
+    ATM  = [EAM, SATM, SCREAM]
```

Comment:
Does not matter for the PR. Would EAMXX be more appropriate here?

Reply:
Yes it would. To be addressed in a future PR.